task_2/longestWordInString.js

```diff
@@ -1 +1,15 @@
+function longestWordInString(str){
+    
+    var reci=str.split(' ');
 
+    var najduzaRec=0;
+
+    for (var i = 0; i < reci.length; i++) {
```

Comment:
ovo je skroz ok, ali evo primer kako moze malo krace da se uradi koriscenjem gotovih js funkcija
```suggestion
 const najduzaRec = reci.reduce((longest, currentWord) =>
    (currentWord.length > longest.length)? currentWord : longest
  , '');
  return najduzaRec.length;
```